draft-ietf-quic-recovery.md

```diff
@@ -167,8 +168,8 @@ of frames contained in a packet affect recovery and congestion control logic:
   performance of the QUIC handshake and use shorter timers for
   acknowledgement.
 
-* Packets that contain only ACK frames do not count toward congestion control
-  limits and are not considered in-flight.
+* Packets that contain only ACK or CONNECTION_CLOSE frames do not count toward
```

Comment:
It's not clear if the "only" applies only to ACKs or to both.

Reply:
@ianswett Thank you for the suggestion. Applied in f7eed75f.